release/ray_release/config.py

```diff
@@ -47,10 +58,45 @@ def read_and_validate_release_test_collection(
 ) -> List[Test]:
     """Read and validate test collection from config file"""
     with open(config_file, "rt") as fp:
-        test_config = yaml.safe_load(fp)
-
-    validate_release_test_collection(test_config, schema_file=schema_file)
-    return test_config
+        tests = parse_test_definition(yaml.safe_load(fp))
+
+    validate_release_test_collection(tests, schema_file=schema_file)
+    return tests
+
+def test_definition_invariant(
+    test_definition: TestDefinition,
+    invariant: bool,
+    message: str
+) -> None:
+    if invariant:
+        return
+    raise ReleaseTestConfigError(
+        f'{test_definition["name"]} has invalid definition: {message}',
+    )
+    
+def parse_test_definition(test_definitions: List[TestDefinition]) -> List[Test]:
+    tests = []
+    for test_definition in test_definitions:
+        if "variations" not in test_definition:
+            tests.append(test_definition)
+            continue
+        variations = test_definition.pop("variations")
+        test_definition_invariant(
+            test_definition, 
+            variations,
+            'empty variations cannot be empty',
```

Comment:
I don't understand this error message, can you make it a bit more clear and maybe actionable? What can the developer do to fix this issue?

Reply:
A dummy error message for sure